docs/tutorials/substreams/substreams-app-example.experimental.md

```diff
@@ -0,0 +1,215 @@
+---
+title: 'Substreams application example'
+
+order: -9999
+outline: [0,4]
+---
+
+# Substreams Application Example
+
+If you follow the deployment of `substreams charts` from [Getting Started](getting-started.experimental.md), you should now have a working `Substreams` cluster locally.
+
+However, this local cluster will only help you to extract block data from `producer` pod or a blockchain if you connect to a public endpoint instead.
+
+To fully make use of these block data you will need something called `Substreams Packages` which will be used like a filter to extract exactly the data you need. For example:
+- Get transaction data from a block
+- Find action data based on action name and account
+- Find transaction by hash
+- etc...
+
+Then with these filtered data, you can use it by storing it in a database, displaying it in an explorer, etc...
+
+You can write your own [Substreams package](https://substreams.streamingfast.io/#develop-substreams), or you can go to [Pinax Substreams package](https://github.com/pinax-network/substreams) for Ultra compatible `Substreams` package.
+
+## Prerequisites
+
+Here is an example that demonstrates how you can make use of `Substreams` package and extract `eosio.token` `transfer` action from `Substreams` cluster.
+
+The example will be a `NodeJS` application written in `Typescript` with `yarn` as package manager, so make sure you have these installed before starting.
```

Comment:
Usually should be `Node.js` , as you see when you open the below links.

Reply:
updated